setup.py

```diff
@@ -42,6 +42,7 @@
     'pytest-cov>=1.8.0',
     'pytest-pep8>=1.0.6',
     'pytest>=2.8.0',
+    'responses>=0.5.1'
```

Comment:
Yes I learned that after choosing `response`. I wanted to use HTTPretty too at first but then I saw this: `HTTPretty is not supporting python3 officially. You will notice that the travis build for python 3 might be broken, and while pull requests fixing py3 support are most welcome, it is still not official at least for now.` https://github.com/gabrielfalcao/HTTPretty